### PR TITLE
Add SVG wordmark copy actions feature in raycast svgl extension

### PR DESCRIPTION
**Pull request by ridemountainpig**
## Description
- Add SVG wordmark copy actions.
- Fix SVG error in multiple categories.

## Screencast
CleanShot 2024-03-19 at 20 05 51@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by ridemountainpig** [comment]
🔔 @1weiho, @pheralb you might want to have a look for the new wordmark actions feature in Raycast Svgl extension.

File: extensions/svgl/package.json [hunk 1]
```diff
@@ -4,7 +4,10 @@
   "title": "Svgl",
   "description": "Search SVG logos via svgl",
   "icon": "command-icon.png",
-  "author": "davidho0403",
+  "author": "1weiho",
```
Comment (pernielsentikaer) [inline extensions/svgl/package.json:7]: Why did you change author?

Comment (ridemountainpig) [reply extensions/svgl/package.json:7]: The author has already named [1weiho](https://www.raycast.com/1weiho) in Raycast but didn't update it in the extension, causing a pull request action check error. Therefore, I changed the author's name from **davidho0403** to **1weiho**.

Comment (ridemountainpig) [reply extensions/svgl/package.json:7]: CleanShot 2024-03-19 at 20 43 13@2x

Comment (pernielsentikaer) [reply extensions/svgl/package.json:7]: Makes sense, thanks for the clarification 🙂

**Comment by 1weiho** [comment]
LGTM👍 Thanks for helping correct the author's name.

**Comment by pheralb** [comment]
Thanks @ridemountainpig @1weiho 🙏🥹✨

**Review by pernielsentikaer — APPROVED** [review]
